example/lib/main.dart

```diff
@@ -1,4 +1,6 @@
 import 'package:android_plugin/android_plugin.dart';
+import 'package:android_plugin_example/utils/numeric_constants.dart';
```

Comment:
check if you can use relative imports here

Reply:
Fixed